src/implementations/twig/components/message/message.html.twig

```diff
@@ -53,7 +53,7 @@
     extra_classes: 'ecl-message__icon',
   } only %}
   {% if _close.icon is defined and _close.icon is not empty %}
-    {% set _icon = _close.icon|merge({ size: 's', name: 'close' }) %}
+    {% set _icon = _close.icon|merge({ name: 'close-filled' }) %}
```

Comment:
We should add a line in the migration doc to say that the size of the icon is no longer set by default in the template; this may need some action on integration side